maputil/maputil.go

```diff
@@ -2,10 +2,11 @@
 package maputil // import "github.com/teamwork/utils/maputil"
 
 // Reverse the keys and values of a map.
-func Reverse(m map[string]string) map[string]string {
-	n := make(map[string]string)
+func Reverse[T comparable, V comparable](m map[T]V) map[V]T {
```

Comment:
`Reverse` should probably be renamed.  I'd argue for `Swap` or something similar.

Reply:
Agreed, when I seen reverse I had no idea what it did until I read the code, and I'm fairly sure that's a bad sign :thinking: